documentation/asciidoc/topics/proc_install_command_line.adoc

```diff
@@ -5,25 +5,21 @@ As an alternative to installing {ispn_operator} through the **OperatorHub** on {
 .Prerequisites
 
 * Have an `oc` client.
+* Create a project for {ispn_operator}.
```

Comment:
I understand **Prerequisites** as something user is expected to already have, while create is a task that should be part of the procedure